Libraries/LibWeb/Namespace.h

```diff
@@ -11,18 +11,11 @@
 
 namespace Web::Namespace {
 
-#define ENUMERATE_NAMESPACES                                            \
-    __ENUMERATE_NAMESPACE(HTML, "http://www.w3.org/1999/xhtml")         \
-    __ENUMERATE_NAMESPACE(MathML, "http://www.w3.org/1998/Math/MathML") \
-    __ENUMERATE_NAMESPACE(SVG, "http://www.w3.org/2000/svg")            \
-    __ENUMERATE_NAMESPACE(XLink, "http://www.w3.org/1999/xlink")        \
-    __ENUMERATE_NAMESPACE(XML, "http://www.w3.org/XML/1998/namespace")  \
-    __ENUMERATE_NAMESPACE(XMLNS, "http://www.w3.org/2000/xmlns/")
-
-#define __ENUMERATE_NAMESPACE(name, namespace_) extern FlyString name;
-ENUMERATE_NAMESPACES
-#undef __ENUMERATE_NAMESPACE
-
-void initialize_strings();
+FlyString const HTML = "http://www.w3.org/1999/xhtml"_fly_string;
```

Comment:
Er sorry, I was not asking you to make this change - I've actually already done that locally for all of the strings initialized from MainThreadVM. More importantly, defining these like this in the header isn't right, as they will be initialized by every file that #includes it. So this change can be reverted.

Reply:
Ah, ok. What change am I supposed to make then?
Or should I just put this on hold until your change has landed?

Reply:
You can leave the hard-coded copy of the HTML namespace string (`"http://www.w3.org/1999/xhtml"sv) in InspectorClient.cpp, I'll take care of removing it when I open a PR with my string changes. Sorry for the confusion, my comment before was just to explain why the string ended up empty 😅

Reply:
Alright, done that now.
Thanks for explaining it though!